data/ecosystems/s/starknet.toml

```diff
@@ -379,13 +379,16 @@ url = "https://github.com/amanusk/starknet-foundry-template"
 url = "https://github.com/amanusk/starknet_token_remover"
 
 [[repo]]
-url = "https://github.com/ametel01/No-Game"
+url = "https://github.com/ametel01/nogame-app"
 
 [[repo]]
-url = "https://github.com/ametel01/starknet-proof-of-attendance"
```

Comment:
why is this URL being removed?

Reply:
The repo is outdated and not maintaned, not worth mentioning i believe

Reply:
we keep all repos so we have their history tracked too and can accurately represent data in the report

Reply:
no problem, i reverted and added the missing repos

Reply:
ty!